src/control/linear_motor.hpp

```diff
@@ -0,0 +1,73 @@
+//
+// Created by 임정훈 on 25. 7. 7.
+//
+
+#ifndef CLINEAR_ACTU_HPP
+#define CLINEAR_ACTU_HPP
+
+#pragma once
+#include "AXL.hpp"
+#include "AXM.hpp"
+#include "stddef.h"
+#include <conio.h>
+#include <iostream>
+#include "../constants.hpp"
+
+char mot_file[] = "hingrui.mot"; // *.mot file Path
+int vel = LINEAR_MOTOR_VEL;
+int accel = LINEAR_MOTOR_ACCEL;
+
+class linear_motor
```

Comment:
```suggestion
class LinearMotor
```

클래스 이름은 snake_case 아님